src/main/java/com/rackspace/ceres/app/services/DataWriteService.java

```diff
@@ -43,22 +46,27 @@
   private final TimeSlotPartitioner timeSlotPartitioner;
   private final DownsampleTrackingService downsampleTrackingService;
   private final AppProperties appProperties;
+  private ReactiveRedisTemplate<String, List<String>> reactiveRedisTemplate;
+
+  private static final String LABEL_METRIC_GROUP = "metricGroup";
 
   @Autowired
   public DataWriteService(ReactiveCqlTemplate cqlTemplate,
-                          SeriesSetService seriesSetService,
-                          MetadataService metadataService,
-                          DataTablesStatements dataTablesStatements,
-                          TimeSlotPartitioner timeSlotPartitioner,
-                          DownsampleTrackingService downsampleTrackingService,
-                          AppProperties appProperties) {
+      SeriesSetService seriesSetService,
```

Comment:
I guess you formatted this. it's better if you use this sparingly so it's more clear to the reviewer which lines actually changed.